aiida_wannier90/workflows/minimal.py

```diff
@@ -262,7 +262,6 @@ def run_w90_pp(self):
 
     def run_pw2wan(self):
         """Run pw2wannier90.x."""
-        self.out('nscf_output', self.ctx.pw_nscf.outputs.output_parameters)
```

Comment:
Why was the `nscf_output` removed?

Reply:
Just because it is repeated, details here https://github.com/aiidateam/aiida-wannier90/pull/96

Reply:
Ah, thanks! I will merge this, then.